assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py

```diff
@@ -268,9 +276,23 @@ def download_model(self):
             )
 
 
-def download_model(model_source: str, model_id: str, download_dir: Path):
+def download_model(model_source: str, model_id: str, download_dir: Path, token: str):
     """Download model and return model information."""
     if model_source == ModelSource.HUGGING_FACE.value:
+        if token:
+            try:
+                # Command to set the 'store' credential helper as default, as no helper is defined by default.
+                cmd = "git config --global credential.helper store"
+                exit_code, stdout = run_command(cmd)
+                if exit_code != 0:
+                    raise AzureMLException._with_error(
```

Comment:
Sure, let me handle it in better way. Thankyou :)

Reply:
Instead of having two try catches, I am moving cmd outside try catch and its respective error if it fails.
Keeping login part inside try catch.
Since we have static command, I dont think keeping it in try catch really makes sense..even if it will fail we will get exitcode != 0 and error we will be able to catch.